File: cesta/AMarotoChivite-cesta/AMarotoChivite-cesta-Java/src/test/java/simulacionCesta/factories/FinalCestFactoryTest.java
package simulacionCesta.factories;

import org.junit.jupiter.api.Test;
import simulacionCesta.models.FinalCest;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class FinalCestFactoryTest {

    @Test
    public void finalCestFactoryCreateTest() {
        FinalCest finalCest = FinalCestFactory.create();
        // Comprobar que la cesta final contiene una cesta no vacía
        assertNotNull(finalCest);
    }
}

File: cesta/AMarotoChivite-cesta/AMarotoChivite-cesta-Kotlin/src/test/kotlin/simulacionCesta/factories/FinalCestFactoryTest.kt
package simulacionCesta.factories

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class FinalCestFactoryTest {
    @Test
    fun finalCestFactoryCreateTest() {
        val finalCest = FinalCestFactory.create()
        // Comprobar que la cesta final contiene una cesta no vacía
        Assertions.assertNotNull(finalCest)
    }
}